ui/src/components/NodeListTable.js

```diff
@@ -342,17 +342,17 @@ const NodeListTable = (props) => {
           // yellow for status.conditions['Ready'] == True and some other conditions are true
           // red for status.conditions['Ready'] == False
           // grey when there is no status.conditions
-          const otherConditions = conditions?.filter(
-            (cond) => cond !== 'ready',
+          const notReadyConditions = conditions?.filter(
+            (cond) => cond !== 'ready' ?? [],
           );
```

Comment:
Not like this, or you'll get an array of arrays :stuck_out_tongue: 

```suggestion
          const notReadyConditions = conditions?.filter(
            (cond) => cond !== 'ready',
          )  ?? [];
```